community/docker/APKBUILD

```diff
@@ -36,7 +37,7 @@ source="
 	runc-$_runc_ver.tar.gz::https://github.com/opencontainers/runc/archive/$_runc_ver.tar.gz
 	containerd-$_containerd_ver.tar.gz::https://github.com/docker/containerd/archive/$_containerd_ver.tar.gz
 	libnetwork-$_libnetwork_ver.tar.gz::https://github.com/docker/libnetwork/archive/$_libnetwork_ver.tar.gz
-
+	tini-$_tini_ver.tar.gz::https://github.com/krallin/tini/archive/$_tini_ver.tar.gz
```

Comment:
@ncopa I can try adding it as a dependency. Who would own the symlink? Is there a clean way to do this? I could also try to patch in the tini path instead of the symlink name.